src/main/java/org/fundacionjala/coding/dennis/Encrypto.java

```diff
@@ -0,0 +1,33 @@
+package org.fundacionjala.coding.dennis;
+
+/**
+ *
+ */
+public interface Encrypto {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Move constants to a class or enum. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1214)